### PR TITLE
Remove default export sizeLimit of 1GB

### PR DESCRIPTION
**Pull request by sedroche**
I am removing the size limit as part of this feature will be to export larger collections.
The limit is still configurable if needed but a falsey value will set no limit

### CODE REVIEW

File: src/endpoints/http/collections/export/index.js [hunk 1]
```diff
@@ -87,6 +87,35 @@ function exportZip(zipFile, out) {
...
+ * @returns {String} - Human readable size with units.
+ */
+function formatBytes(bytes) {
+  if (bytes === 0) {
+    return '0 Bytes';
+  }
+
+  const k = 1000;
+  const dm = 2;
+  const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
```
Comment (pb82) [inline src/endpoints/http/collections/export/index.js:113]: Are you sure that Yottabyte is enough? What if someones database is larger than one trillion terrabytes?

File: config/dev.json [hunk 2]
```diff
@@ -26,7 +26,7 @@
     "cluster/reseller/customer/domain/project/cloud-apps/data-browser"
   ],
   "port": 8822,
-  "sizeLimit": 524288000,
+  "sizeLimit": 1000000000,
```
Comment (pb82) [inline config/dev.json:29]: The description says that the 1Gb sizeLimit is removed but it looks like it's still in use?

File: src/endpoints/http/collections/export/index.js (hunk 1 above)
Comment (sedroche) [reply src/endpoints/http/collections/export/index.js:113]: :laughing: just future proofing....

File: config/dev.json (hunk 2 above)
Comment (sedroche) [reply config/dev.json:29]: We still have a configurable limit though so we need to test it. 1GB should be enough for local dev. I have set fhcap and osePlaceholders values to zero

**Review by pb82 — APPROVED** [review]
LGTM

**Pull request merged**